app.js

```diff
@@ -251,7 +251,11 @@ const userProfile = () => {
 
             if (parameters?.token) {
                 const response = await validateToken(parameters.token); // Add uid and sign-in flag if token is valid
-                if (response.code === 200) {
+                if (response.code === 202 && response.message === 'Duplicate account') {
```

Comment:
would 409 be a better code than 202.

https://metamug.com/article/status-code-409.html#using-409-for-duplicate-record

Reply:
The reason to use a 'success' code is to avoid sending un-necessary errors to Datadog, as suggested by Tony [here](https://github.com/episphere/connect/issues/638).

In addition, a records is labeled as 'duplicate' by site before participant access, and the PR enables us blocking participant to update it. So it's not the exact scenario as described in code 409.

Reply:
Good to know thanks!